rotkehlchen/exchanges/coinbase.py

```diff
@@ -1042,8 +1048,23 @@ def _deserialize_history_event(self, raw_data: dict[str, Any]) -> HistoryEvent |
             asset = asset_from_coinbase(amount_data['currency'], time=timestamp)
             notes = raw_data.get('details', {}).get('header', '')
             tx_type = raw_data['type']
+            if tx_type in ('staking_transfer', 'unstaking_transfer'):
+                event_subtype = HistoryEventSubType.DEPOSIT_ASSET if tx_type == 'staking_transfer' else HistoryEventSubType.REMOVE_ASSET  # noqa: E501
+                return HistoryEvent(
+                    event_identifier=f'{CB_EVENTS_PREFIX}{raw_data["id"]!s}',
+                    sequence_index=0,
+                    timestamp=ts_sec_to_ms(timestamp),
+                    location=Location.COINBASE,
+                    event_type=HistoryEventType.STAKING,
+                    event_subtype=event_subtype,
+                    asset=asset,
+                    balance=Balance(amount),
+                    location_label=None,
```

Comment:
For location label in exchanges we use the exchange name. `self.name`. Which you can even see in the same file.
```suggestion
                    location_label=self.name,
```